### ISSUE TITLE
Error: local variable 'times' referenced before assignment

### ISSUE THREAD

**Issue by dvagala** [ThereforeGames/unprompted#41]
Hey, thanks for a great extension!

I'm experiencing an error though when running the basic example `[file human/main]` in AUTOMATIC1111.

```
Traceback (most recent call last):
  File "/Users/dvagala/temp/stable-diffusion-webui/extensions/unprompted/lib/shortcodes.py", line 133, in render
    return str(self.handler(self.token.keyword, self.pargs, self.kwargs, context, content))
  File "/Users/dvagala/temp/stable-diffusion-webui/extensions/unprompted/lib/shared.py", line 57, in handler
    return(self.shortcode_objects[f"{keyword}"].run_block(pargs, kwargs, context, content))
  File "/Users/dvagala/temp/stable-diffusion-webui/extensions/unprompted/shortcodes/basic/choose.py", line 46, in run_block
    for x in range(0, times):
UnboundLocalError: local variable 'times' referenced before assignment
```

Python 3.9.12
Apple M1 Pro
MacOs Venture 13.0

**Comment by dvagala** [comment]
I fixed it locally just by setting `times = 1` in `choose.py` in advance, but I have no idea what it is doing, and why

**Comment by ThereforeGames** [comment]
Fixed in v4.3.1, thank you @dvagala!
